### PR TITLE
Fluentd tuning

### PR DESCRIPTION
**Pull request by djsd123**
Elasticsearch has been degraded up until recently due to high memory pressure caused by slow garbage collection. 
Leading up to this FluentD has been constantly logging exceptions mainly related to 

* connection issues with elasticsearch 
and 
* continuously exceeding buffer limits. partially due to misconfiguration  

Trial and error while parsing docs has lead me to what looks like a semi-stable configuration.

## Test `dev`
1. Get a pod id to parse logs from
`kubectl get po --namespace kube-system | grep logging`

2. Parse the logs to hopefully see a reduced frequency of exceptions
`kubectl logs < POD ID > --namespace kube-system`

### CODE REVIEW

File: charts/fluentd/templates/clusterrole.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     app: {{ template "fullname" . }}
     kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```
Comment (xoen) [inline charts/fluentd/templates/clusterrole.yaml]: What is it?

Comment (djsd123) [reply charts/fluentd/templates/clusterrole.yaml]: See commit: https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/136/commits/cfeb5d70ed8fc406427c479717a68324c24f3368

File: charts/fluentd/templates/daemonset.yaml [hunk 2]
```diff
@@ -64,7 +71,10 @@ spec:
           mountPath: /var/lib/docker/containers
           readOnly: true
         - name: fluentdconf
-          mountPath: /fluentd/etc
+          mountPath: /etc/fluent/config.d
+        - name: libsystemddir
```
Comment (xoen) [inline charts/fluentd/templates/daemonset.yaml]: What is this? Why do we need to mount `/usr/lib64`?

File: charts/fluentd/values.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 image:
-  repository: quay.io/mojanalytics/fluentd # Temporary until we can get upstream working again See: https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/127
-  tag: tempFork
+  repository: gcr.io/google-containers/fluentd-elasticsearch
```
Comment (xoen) [inline charts/fluentd/values.yaml]: 🎊

File: charts/fluentd/templates/daemonset.yaml (hunk 2 above)
Comment (djsd123) [reply charts/fluentd/templates/daemonset.yaml]: :-P, lol See commit: https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/136/commits/cb2de52bb1212d5967c06e622cf0568b08c98754

**Comment by djsd123** [comment]
Thanks @xoen

**Pull request merged**